src/components/global/Playground/stackblitz.utils.ts

```diff
@@ -0,0 +1,125 @@
+import sdk from '@stackblitz/sdk';
+
+// The default title to use for Stackblitz examples (when not overwritten)
+const DEFAULT_EDITOR_TITLE = 'Ionic Docs Example';
+// The default description to use for Stackblitz examples (when not overwritten)
+const DEFAULT_EDITOR_DESCRIPTION = '';
+// Default package version to use for all @ionic/* packages.
+const DEFAULT_IONIC_VERSION = '^6.0.0';
+
+interface EditorOptions {
+  /**
+   * The title of the Stackblitz example.
+   */
+  title?: string;
+  /**
+   * The description of the Stackblitz example.
+   */
+  description?: string;
+}
```

Comment:
Could we export this and use it as the type for `editorOptions` in `openEditor()` from `Playground.tsx`?